pyproject.toml

```diff
@@ -20,7 +20,7 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = '>=3.7.2,<3.11'
+python = '>=3.7.2,<3.12'
```

Comment:
Maybe `<=3.11` makes more sense for the purpose?

Reply:
Done